### PR TITLE
Update UI for the Cymbal Shops Rebrand

### PR DESCRIPTION
**Pull request by NimJay**
See the "What needs to be done?" section of #574.

**Change Summary**
* This pull-request updates all pages of "Online Boutique".
* This pull-request applies UI changes from the new [Figma prototypes](https://www.figma.com/file/UlhTLEvRFIw8vK5wAzD7Vq/OnlineBoutique.dev?node-id=6%3A14) (only accessible to Googlers with a Figma subscription) for the new "Cymbal Shops" redesign.

**Additional Notes**
* Changes related to the _Cymbal Shops_ logo and the product list update will happen in a different pull-request. 
* This designs from this pull-request will be reviewed by the Google Cloud Demos and Experiments team designers.

### CODE REVIEW

File: src/frontend/static/icons/Hipster_CartIcon.svg
Comment (NimJay) [inline src/frontend/static/icons/Hipster_CartIcon.svg:1]: The `Hipster_CartIcon.svg` file contains changes because the top-right cart icon has a new look.

File: src/frontend/handlers.go
Comment (NimJay) [inline src/frontend/handlers.go:479]: The `renderMoney` function was altered because we no longer spell out the currency — we use the currency symbol instead (e.g., "USD" is not "$").

File: src/frontend/static/icons/Hipster_DownArrow.svg
Comment (NimJay) [inline src/frontend/static/icons/Hipster_DownArrow.svg:1]: The `Hipster_DownArrow.svg` file is the image used as the down chevron/arrow in drop-downs (e.g., see top-right currency drop-down). It looks different now.

File: src/frontend/static/icons/Hipster_NavLogo.svg
Comment (NimJay) [inline src/frontend/static/icons/Hipster_NavLogo.svg:1]: The `Hipster_NavLogo.svg`'s "Online Boutique" text is not black — not white (see top-right logo of all pages).

File: src/frontend/static/images/credits.txt
Comment (NimJay) [inline src/frontend/static/images/credits.txt:2]: I've created a new credits.txt for the two new images we got from unsplash.com — copying the format of the [existing credits.txt file](https://github.com/GoogleCloudPlatform/microservices-demo/blob/release/v0.2.4/src/frontend/static/img/products/credits.txt).

**Review by bourgeoisor — APPROVED** [review]
This LGTM.

Minor, but I noticed that there is a text highlight / background that is sliiiiightly of a different tone than the background-background to the point where it hurts my eyes. Is that intentional? I assumed they should be the exact same color.

See screenshot below:
image

**Comment by NimJay** [comment]
Thanks so much for the review, @bourgeoisor!

**1. Cart Field Background Color**
> Minor, but I noticed that there is a text highlight / background that is sliiiiightly of a different tone than the background-background to the point where it hurts my eyes. Is that intentional?

Good eye! You're right, the background _is_ different — and yes, this is intention (i.e., matches the Figma prototypes).
Luckily, the designers have changed (darkened) the background of each page, so that this contrast looks less like an accident. 😌

Screen Shot 2021-09-02 at 1 34 29 PM 

**2. Designers' Feedback**
Last week (Aug 26), the designers provided feedback (see [Google-internal doc](https://docs.google.com/document/d/1mrX9MPITnWhR-b-VeZUqTZTZvA9mF71uHGVmgkWOMoY/edit?resourcekey=0-xl19Ety9CS-rPA6nGVUn8g)) and improved a few things.
I have implemented those changes (see recent commits in this PR).
Thought I'd let you know in case you're wondering why the staged changes look different. :)

**Comment by bourgeoisor** [comment]
My LGTM still stands, code-wise. Let me know if you want me to re-review something!

**Comment by NimJay** [comment]
These UI changes have been approved by the Cloud Experiments & Demos designers ([relevant internal meeting notes](https://docs.google.com/document/d/1mrX9MPITnWhR-b-VeZUqTZTZvA9mF71uHGVmgkWOMoY/edit?resourcekey=0-xl19Ety9CS-rPA6nGVUn8g)).
I will talk to @askmeegs to figure out the right timing for this merge. Ideally, we'd merge this right after a releasing v0.2.5.

**Review by askmeegs — APPROVED** [review]

**Comment by NimJay** [comment]
@askmeegs and I decided **not** to create a release **before** merging these UI changes, since the commits from the previous release (v0.2.4 in Jul 15, 2021) aren't urgent fixes/changes.
I'm merging this now.

**Pull request merged**
